spec/ensure_consistency_spec.rb

```diff
@@ -41,5 +41,8 @@
 
     two_vote_topic.reload
     expect(two_vote_topic.topic_vote_count.votes_count).to eq(2)
+
+    user2.destroy
+    expect { second_vote.reload }.to raise_error(ActiveRecord::RecordNotFound)
```

Comment:
I suspect it was added because activerecord dependent:destroy hooks are not guaranteed to fire, especially since we don't make consistent use of transactions.

But anyway I agree this is probably patching up some old bad data, and we can try without 👌